app/assets/stylesheets/_forms.scss

```diff
@@ -0,0 +1,181 @@
+.form-box {
+  padding: 0 $spacing-unit $spacing-unit $spacing-unit;
+  overflow: auto;
+
+  background: #FFFFFF;
+  border-radius: $ui-border-radius;
+
+  h2 {
+    @include font-size-heading(1, $font-size-h2);
+
+    width: 100%;
+
+    text-align: center;
+    color: $color-gray;
+  }
+
+  form {
+    margin: 0;
+  }
+
+  #error_explanation {

```

Comment:
leave it it's only few days to the end, better focus all power to finish product on time, it's not important